app/controllers/apo_controller.rb

```diff
@@ -87,8 +87,16 @@ def set_apo_metadata apo, md_info
     apo.metadata_source      = md_info[:metadata_source]
     apo.agreement            = md_info[:agreement].to_s
     apo.default_workflow     = md_info[:workflow ] unless (!md_info[:workflow] || md_info[:workflow].length < 5)
-    apo.use_license          = md_info[:use_license]
     apo.default_rights       = md_info[:default_object_rights]
+    if md_info[:use_license].blank?
+      # delete use licenses, ensure defaultObjectRights datastream is saved correctly
+      apo.defaultObjectRights.creative_commons = nil
+      apo.defaultObjectRights.open_data_commons = nil
+      apo.defaultObjectRights.creative_commons_human = nil
```

Comment:
i'm not so opposed to storing the human readable string, but i do think that we should be using one unified setter method that takes something like a license code, and sets URI and human readable based on that.  i think we should consider setting those things individually to be deprecated, at best.